source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -99,111 +106,104 @@ deployment and your security needs.
 
          On systems with limited resources, some performance degradation may
          be observed while swap is disabled.
-         
-#. Create (or edit if it exists) the hosts file that kubernetes will read to 
-   locate master's host:
+
+#. Switch to root to modify `hostname`:
 
    .. code-block:: bash
 
-      echo "127.0.0.1 localhost `hostname`" >> /etc/hosts
+      sudo -s
 
-#. Configure the Kubernetes runtime interface, either:
+#.  Create (or edit if it exists) the hosts file that Kubernetes will read to
+    locate the master's host:
 
-   a) Run Kubernetes with Docker + runc:
+    .. code-block:: bash
 
-      #. Enable docker.service:
+       echo "127.0.0.1 localhost `hostname`" >> /etc/hosts
 
-         .. code-block:: bash
+#.  Exit root:
 
-            sudo systemctl enable docker.service
+    .. code-block:: bash
 
-      #. Create (or edit if it exists) the file 
-         :file:`/etc/systemd/system/docker.service.d/51-runtime.conf` and include the following lines:
+       exit
 
-         .. code-block:: bash
+Configure and run Kubernetes
+****************************
 
-            [Service]
-            Environment="DOCKER_DEFAULT_RUNTIME=--default-runtime runc"
+This section describes how to configure and run Kubernetes with:
 
-      #. Create (or edit if it exists) the file :file:`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` and include the following lines:
+* Docker and runc
+* CRI-O and kata-runtime
 
-         .. code-block:: bash
+Configure and run Docker + runc
+===============================
 
-            [Service]
-            Environment="KUBELET_EXTRA_ARGS="
+#.  Enable the Docker service:
 
-      #. Restart services:
+    .. code-block:: bash
 
-         .. code-block:: bash
+       sudo systemctl enable docker.service
```

Comment:
This can be eliminated since Docker is started at the beginning.

Reply:
Agreed to eliminate docker commands at start per discussion here.